### PR TITLE
common: target optee-os-devkit builds the devkit

### PR DESCRIPTION
**Pull request by etienne-lms**
Adds build target `optee-os-devkit` to get the TA devkit be built and installed. It can be later used as a dependency when building external TAs that need to build linked to OP-TEE as early TAs where building OP-TEE core depends on such TAs being already built. This applies to ftpm hence replace the dependency of `ftpm` make target with a dependency on `optee-os-devkit`.

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -568,7 +572,7 @@ FTPM_FLAGS ?= 						\
 .PHONY: ftpm
 ftpm:
 ifeq ($(MEASURED_BOOT_FTPM),y)
-ftpm: optee-os
+ftpm: optee-os-devkit
```
Comment (jforissier) [inline common.mk:575]: This dependency looks good, but I don't see where/how the ftpm TA is linked into OP-TEE as an early TA. Perhaps this extra information should be omitted from the commit description.

**Review by jforissier — COMMENTED** [review]
One comment, then:

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
With Jerome's comment addressed:
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: common.mk (hunk 1 above)
Comment (etienne-lms) [reply common.mk:575]: I was considering one building with `EARLY_TA_PATHS=/path/to/ftpm/build/bc50d971-d4c9-42c4-82cb-343fb7f37896.stripped.elf`.

Nevertheless, I'll remove the related information in the commit message.

**Comment by etienne-lms** [comment]
I've removed the pointed sentence from the commit message and slightly rephrased it.
Review tags applied.

**Pull request merged**